app/src/main/java/ch/epfl/favo/view/tabs/favorList/FavorPage.java

```diff
@@ -34,7 +34,7 @@
 import ch.epfl.favo.R;
```

Comment:
We should rename this class to MyFavorsPage

Reply:
Done